fv3core/stencils/remapping_part1.py

```diff
@@ -23,21 +36,77 @@ def init_pe2(pe: sd, pe2: sd, ptop: float):
 
 
 @gtstencil()
-def delz_adjust(delp: sd, delz: sd):
+def undo_delz_adjust(delp: FloatField, delz: FloatField):
```

Comment:
Does this get used anywhere? Or just `undo_delz_adjust_and_copy_peln`?

Reply:
Nice catch!